target/scripts/helpers/utils.sh

```diff
@@ -105,15 +122,16 @@ function _replace_by_env_in_file() {
   fi
 
   local ENV_PREFIX=${1} CONFIG_FILE=${2}
+  local KV_DELIMITER=${3:-'='}
```

Comment:
We normally don't quote here. Any reason to quote the equal sign?

Reply:
> Any reason to quote the equal sign?

Probably because `:-=` and `:=` are easy to mix up? 😅 

If you and @georglauterbach would like the quote-wrapping dropped to make the string value less distinct, I can do that. I just don't think it's helpful towards contributors that aren't as well-versed with bash and may confuse it as a bashism 🤷‍♂️

Reply:
I actually see the point here, and I am fine with keeping it. @casperklein is riht though,; we should not start quoting in other places.